### PR TITLE
Fix channel_updated WS event

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: app/components/channel_drawer/channel_drawer.js [hunk 1]
```diff
@@ -125,6 +127,14 @@ export default class ChannelDrawer extends PureComponent {
         };
     };
 
+    handleUpdateTitle = (channel) => {
+        let channelName = '';
+        if (channel.display_name) {
+            channelName = channel.display_name;
+        }
+        this.props.actions.setChannelDisplayName(channelName);
```
Comment (hmhealey) [inline app/components/channel_drawer/channel_drawer.js]: This will change the display name when other channels are updated, not just the current one

Comment (enahum) [reply app/components/channel_drawer/channel_drawer.js]: The event is only fired when the updsted channel is the current channel.

The websocket does not use the action as this is an action defined in the client and not available in redux

Comment (enahum) [reply app/components/channel_drawer/channel_drawer.js]: See https://github.com/mattermost/mattermost-redux/blob/master/src/actions/websocket.js#L479

Comment (hmhealey) [reply app/components/channel_drawer/channel_drawer.js]: Alright, I guess that'll work, but I think we need to consider adding a more general purpose mechanism to have the mobile app dispatch actions in response to websocket events. Using the generic event emitter to go outside of redux to have the ChannelDrawer dispatch an event in response feels hacky.

**Comment by hmhealey** [comment]
@enahum Needs a rebase

**Pull request merged**